XRTK-Core/Packages/com.xrtk.core/Definitions/MixedRealityRegisteredServiceProvidersProfile.cs

```diff
@@ -10,12 +10,12 @@ namespace XRTK.Definitions
     public class MixedRealityRegisteredServiceProvidersProfile : BaseMixedRealityProfile
     {
         [SerializeField]
-        private MixedRealityServiceConfiguration[] configurations = new MixedRealityServiceConfiguration[0];
+        private MixedRealityExtensionServiceConfiguration[] configurations = new MixedRealityExtensionServiceConfiguration[0];
```

Comment:
Just a note, that even though the type was renamed, this shouldn't affect the serialization as long as the field names remain the same.

This might get tricky later when I try to consolidate and make these serialized configuration profiles generic enough later.